src/charts/helpers/style.js

```diff
@@ -2,6 +2,19 @@ module.exports = (function() {
 
     'use strict';
 
+    const computeDefaultStyleByTagName = (tagName) => {
```

Comment:
Would add comments to this function, there were no comments before but I think it would be helpful.

Reply:
Makes sense!